intranet/apps/eighth/models.py

```diff
@@ -802,20 +803,22 @@ def has_open_passes(self):
         """Return whether there are passes that have not been acknowledged."""
         return self.eighthsignup_set.filter(after_deadline=True, pass_accepted=False).exists()
 
+    def _get_viewable_members(self, user):
+        if user and (user.is_eighth_admin or user.is_eighthoffice or user.is_teacher):
+            return self.members.all()
+        else:
+            q = Q(properties__parent_show_eighth=True, properties__self_show_eighth=True)
+            if user:
+                q |= Q(id=user.id)
+            return self.members.filter(q)
+
     def get_viewable_members(self, user=None):
         """Get the list of members that you have permissions to view.
 
         Returns: List of members
```

Comment:
```suggestion
         Returns: Queryset of members with permission to view
```

Reply:
`get_viewable_members()` does actually return a `list` -- it calls `sorted()`. Would you recommend using `order_by()` instead?

Reply:
I would recommend returning a list (and therefore using order_by).